src/settings/NotificationsScreen.js

```diff
@@ -124,6 +125,13 @@ export default function NotificationsScreen(props: Props): Node {
   const auth = authOfAccount(account);
   const identity = identityOfAccount(account);
   const { ackedPushToken } = account;
+
+  // Borrow the view-model from AccountPickScreen just for the UI that's
+  // meant as a limited preview of that screen, i.e., a warning on the
+  // "Other accounts" row that means some of the non-active accounts have a
+  // problem with notifications.
+  const accountStatuses = useGlobalSelector(getAccountStatuses);
```

Comment:
I'm not sure if this is OK. 😅 Before I rebased atop your #5649, I was doing something like `shouldWarnAboutNotifsInAccountPick` on the globally available `getAccountStatuses` selector, which also seemed not-good.

Reply:
:sweat_smile: Yeah, this seems better than that, though still slightly uncomfortable.

I think the reasoning in this comment makes sense.